### DISCUSSION TITLE
Vaga JR

### DISCUSSION THREAD

**Discussion by Bryanzef** [frontendbr/forum#2392] [Perguntas e Respostas]
Galera, estou indo para o 3 semestre de ADS na faculdade e estou com dificuldade para encontrar vagas, já posso dizer que tenho o conhecimento necessário para vaga Jr, só que o retorno nunca vem, e para vagas presencial na minha cidade não tem nada relacionado a área

**Comment by felquis** [comment]
Primeiramente, boa sorte em sua jornada.

Recomendo dar WATCH nas notificações do repositório de vagas da comunidade https://github.com/frontendbr/vagas 

Todo dia aparece muitas por lá.

Não deixe de olhar no linkedin, remoteok, e outros sites de vagas relevantes.
